src/info_extras.ts

```diff
@@ -223,7 +223,9 @@ export const getRelatedVideos = (info: any) => {
   try {
     rvsParams = info.response.webWatchNextResponseExtensionData.relatedVideoArgs
       .split(",")
-      .map((e: any) => qs.parse(e));
+      .map((e: any) => new URLSearchParams(e))
+      .map(( params ) => [ ... params.entries() ] )
+      .map(( params ) => Object.fromEntries(params) );
```

Comment:
FYI this is an array, not a lazy iterator where chained maps can combine for less time complexity. This is going to iterate over the params `3 * n` times. Just do
```ts
.map((e: any) => Object.fromEntries(new URLSearchParams(e)))
```